mysql/resource_database_test.go

```diff
@@ -106,11 +106,21 @@ func testAccDatabaseCheckFull(rn string, name string, charset string, collation
 			return fmt.Errorf("error reading database: %s", err)
 		}
 
-		if strings.Index(createSQL, fmt.Sprintf("CHARACTER SET %s", charset)) == -1 {
+		if !strings.Contains(createSQL, fmt.Sprintf("CHARACTER SET %s", charset)) {
```

Comment:
Changed at linting recommendation and agree it's easier to read.

Reply:
Thanks. that's fine